packages/boilerplate-generator/template-web.browser.js

```diff
@@ -6,7 +6,18 @@ export const headTemplate = ({
   bundledJsCssUrlRewriteHook,
   head,
   dynamicHead,
-}) => [
+}) => {
+  var headSections = head.split(/<meteor-bundled-css[^<>]*>/, 2);
+  // if (headSections.length > 1) {
```

Comment:
Please remove this commented code.

Reply:
Sorry, thought I had removed this before.